lua/autorun/server/sv_spawn_protection.lua

```diff
@@ -144,52 +143,43 @@ end
 
 -- Function called on player spawn to grant spawn protection
 local function setSpawnProtectionForPvpSpawn( player )
-    if ( playerIsInPvp( player ) ) then
-		if( !playerSpawnedAtEnemySpawnPoint( player ) ) then
-			setSpawnProtection( player )
-			setPlayerTransparent( player )
-			createDecayTimer( player )
-		end
-    end
+	if not playerIsInPvp( player ) then return end
+	if playerSpawnedAtEnemySpawnPoint( player ) then return end
+	
+	setSpawnProtection( player )
+	setPlayerTransparent( player )
+	createDecayTimer( player )
 end
 
 -- Called on weapon change to check if the weapon is allowed,
 -- and remove spawn protection if it's not
 local function spawnProtectionWeaponChangeCheck( player, oldWeapon, newWeapon)
-    if ( playerIsInPvp( player ) ) then
+	if not playerIsInPvp( player ) then return end
+	if not playerHasSpawnProtection( player ) then return end
+	if weaponIsAllowed( newWeapon ) then return end
 
-        if ( playerHasSpawnProtection( player ) ) then
-
-            if ( !weaponIsAllowed( newWeapon ) ) then
-                removeSpawnProtection( player )
-                setPlayerVisible( player )
-                removeDecayTimer( player )
-                removeDelayedRemoveTimer( player )
-            end
-        end
-    end
+    removeSpawnProtection( player )
+    setPlayerVisible( player )
+    removeDecayTimer( player )
+    removeDelayedRemoveTimer( player )
 end
 
 -- Called on player keyDown events to check if a movement key was pressed
 -- and remove spawn protection if so
 local function spawnProtectionMoveCheck( player, keyCode )
```

Comment:
More indentation weirdness in this method

Reply:
Man, I saw that too. 
The vim boy is set to four spaces, though. 
I'll check 'er out

Reply:
`gg=G`